### PR TITLE
fix: :hammer: removed swagger comments from routes file

### PR DESCRIPTION
**Pull request by rajat-mehra05**
Closes: #602 
- removed swagger comments from routes file

### CODE REVIEW

File: routes/pullrequests.js [hunk 1]
```diff
@@ -2,119 +2,7 @@ const express = require("express");
...
- *                    items:
- *                      $ref: '#/components/schemas/pullRequests'
- *       500:
- *         description: badImplementation
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/badImplementation'
- */
-router.get("/stale", pullRequest.getStalePRs);
```
Comment (samarpan1738) [inline routes/pullrequests.js:87]: @rajat-mehra05 Was this intended to be removed?

File: routes/members.js [hunk 2]
```diff
@@ -3,285 +3,18 @@ const router = express.Router();
...
- *
- *       500:
- *         description: serverUnavailable
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/serverUnavailable'
- */
-
-router.get("/intro", authenticate, authorizeUser(SUPER_USER), fetchRecruitersInfo);
```
Comment (samarpan1738) [inline routes/members.js:124]: Was this intended to be removed?

File: routes/tasks.js [hunk 3]
```diff
@@ -2,240 +2,14 @@ const express = require("express");
...
- *           schema:
- *             $ref: '#/components/schemas/tasks'
- *     500:
- *       description: badImplementation
- *       content:
- *         application/json:
- *           schema:
- *             $ref: '#/components/schemas/errors/badImplementation'
- */
-router.post("/", authenticate, authorizeUser("appOwner"), createTask, tasks.addNewTask);
```
Comment (samarpan1738) [inline routes/tasks.js:131]: Was this intended to be removed?

**Review by samarpan1738 — CHANGES_REQUESTED** [review]

File: routes/pullrequests.js (hunk 1 above)
Comment (rajat-mehra05) [reply routes/pullrequests.js:87]: No, I had accidently removed the line. Thanks!

File: routes/members.js (hunk 2 above)
Comment (rajat-mehra05) [reply routes/members.js:124]: No

File: routes/tasks.js (hunk 3 above)
Comment (rajat-mehra05) [reply routes/tasks.js:131]: No.

**Review by pallabez — DISMISSED** [review]
LGTM :+1:

**Review by samarpan1738 — DISMISSED** [review]
LGTM 👍

**Review by pallabez — APPROVED** [review]
Thanks for the changes

**Pull request merged**